handbook/engineering/engineering-management.md

```diff
@@ -54,3 +54,21 @@ Here are some books that multiple people on the team have read and recommend. If
   - Session 3 week of June 21
   - Session 4 week of July 5
   - Session 5 week of July 19
+
+## Temporary Management Positions
+
+We create temporary management positions when there is an organizational need. These may be filled by someone who is transitioning into the role (Interim Manager), experimenting with the role as they work on determining their career path (Acting Manager) or, someone who is just filling in and who is not interested in pursuing an Engineering Manager role long term (Acting Manager). This difference should be made explicit with the individual and team members before the temporary role is assigned.
+
+### Acting Manager
+
+An Acting Manager occupies a role temporarily and will move back to their original position after a set amount of time or other conditions. An Acting Manager may be experimenting with the role as a part of determining their career development path or could be filling in for a vacant role while we hire.
+
+### Interim Manager
```

Comment:
I'm iffy on this section.  I think we should be explicit on the Interim Manager position, that its a trial for x number of days.  Often I have seen someone stuck in "almost manager" position for years, with no end in sight.  If we have a need for a manager role, and we have someone that thinks they can level up into that role, and the appropriate manager AND manager's manager (director or VP depending on the case) agree, we should have a trial, with check-ins along the way


I think something like 6-months is acceptable, with check-ins on "almost manager" every month.  There is an escape shoot (one or both parties decide its not working) and there is a deadline for official manager promotion